packages/core-types/src/smart-account.types.ts

```diff
@@ -9,19 +9,21 @@ import { MultiSendCallOnlyContract } from './contracts/MultiSendCallOnlyContract
 import { SmartWalletContract } from './contracts/SmartWalletContract'
 import { GasLimit } from './transaction.types'
 import { Signer } from 'ethers'
+import { IPaymasterAPI } from 'account-abstraction-types'
 
 export interface SmartAccountConfig {
   activeNetworkId: ChainId // same
   supportedNetworksIds: ChainId[] // Network[] chainId: CbainId, rpcUrl?: string
```

Comment:
Add environment field which can have values "testnet" or "mainnet" and maintain two default values, 1 for testnet and 1 for mainnet. Be default it should be "mainnet"

Reply:
can you share more details? i missed this part of the discussion

Reply:
The default values for the config which we have are of test networks but we should have one default configuration for main networks also.... and to do that we can have one 'environment' field in SmartAccountConfig which can have values "test" or "mainnet".. 

Like we have for [Hyphen](https://docs.biconomy.io/products/hyphen-instant-cross-chain-transfers/sdk#2.-importing-and-instantiation)

So if developer set this field as "test" then we'll use the defaultConfig value which has test networks values.. and vice versa.

Reply:
got it. let me create another PR